packages/gatsby-transformer-xml/README.md

```diff
@@ -13,6 +13,43 @@ Parses XML files. It supports also attributes
 plugins: [`gatsby-transformer-xml`]
 ```
 
+**With options**
+
+```javascript
+// In your gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-transformer-xml`,
+    options: {
+      // Whether to trim whitespace characters that may exist before and after the text.
+      trim: false,
+      // Whether to attempt converting text of numerals or of boolean values to native type. For example, "123" will be 123 and "true" will be true
+      nativeType: false,
```

Comment:
you should mention which are mandatory and which are not ...

Reply:
I can make it more explicit, sure :)
All options are optional - I’ll adjust the docs.

Reply:
thats good - but if all are optional then in the merge part you should check all values against `undefined`